### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by MahdiAghaali**
## In this branch I did the following activities and changes:

- Add basic UI to run the code

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,29 +1,166 @@
...
+    end
 
-puts book1.rental_history
-puts book2.rental_history
-puts person.rental_history
-puts classroom.students
+    @rental.filter do |el|
+      puts "#{el.book.title} (#{el.date})" if el.person == @people[person_index.to_i - 1]
+    end
+  end
```
Comment (TedLivist) [inline app.rb:165]: - [ ] After listing a rental the app exits. The app should still keep running after listing a rental. Kindly fix that.

File: app.rb [hunk 2]
```diff
@@ -1,29 +1,166 @@
...
+
+  def people_list
+    puts 'List of people:'
+    puts ''
+    @people.each_with_index do |el, i|
+      puts "#{i + 1}- #{el.name}"
+    end
+    show_link_to_main
+  end
```
Comment (TedLivist) [inline app.rb:70]: - [ ] When listing the people, kindly list them with their IDs, ages and names as we have on the video demo instead of just the name.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @MahdiAghaali,

Good job so far!
✔️ Good documentation
✔️ Implementation of app UI
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MahdiAghaali** [comment]
@TedLivist Thank you for the feed back

**Review by Reem-lab — APPROVED** [review]
# Hi @MahdiAghaali, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- No Linters errors ✅ 
- Followed Gitflow ✅
- Great working on the functionality🔥🚀🎉
- Professional README ✅



## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**